vpn/BUILD.gn

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
This should go under components/brave_vpn/win

Reply:
done, thanks